pglogical_create_subscriber.c

```diff
@@ -1040,9 +1076,15 @@ initialize_replication_origin(PGconn *conn, char *origin_name, char *remote_lsn)
 	}
 	else
 	{
+		escaped_origin_name = PQescapeLiteral(conn, origin_name, strlen(origin_name));
+		escaped_remote_lsn = PQescapeLiteral(conn, remote_lsn, strlen(remote_lsn));
 		printfPQExpBuffer(query, "INSERT INTO pglogical_origin.replication_origin (roident, roname, roremote_lsn) SELECT COALESCE(MAX(roident::int), 0) + 1, %s, %s FROM pglogical_origin.replication_origin",
-						  PQescapeLiteral(conn, origin_name, strlen(origin_name)),
-						  remote_lsn ? PQescapeLiteral(conn, remote_lsn, strlen(remote_lsn)) : "0");
+						  escaped_origin_name,
+						  remote_lsn ? escaped_remote_lsn : "0");
```

Comment:
If remote_lsn==NULL, we'll have crashed at the earlier strlen(remote_lsn).  Would you account for that?